tests/testthat/test-helpers-simulations.R

```diff
@@ -0,0 +1,36 @@
+test_that("barplot_percentages produces consistent results", {
+  set.seed(504409)
+
+  plot <- crmPack:::barplot_percentages(
```

Comment:
no need to use this here, because tests have access to internal functions
```suggestion
  plot <- barplot_percentages(
```